fido-device-onboard.spec

```diff
@@ -0,0 +1,133 @@
+%define dracutlibdir %{_prefix}/lib/dracut
+%bcond_without check
+%global __cargo_skip_build 0
+%global __cargo_is_lib() false
+%global forgeurl https://github.com/fedora-iot/fido-device-onboard-rs
```

Comment:
It doesn't look like this or the forgemeta below are used anywhere

Reply:
they're used to generate metadata as per fedora/rhel guidelines